ingest/main.go

```diff
@@ -72,13 +86,17 @@ func main() {
 There are two supported datasets: 
   1. 311-service-requests
   2. operating-budget
+  3. ase-tickets
 
 To ingest either one, pass the dataset name as an argument to this program. For example:
   ./ingest 311-service-requests
 
 You can scope to a single year by passing the --year flag. For example:
   ./ingest --year 2017 311-service-requests
 
+You can scope to a single year by passing the --year flag. For example:
```

Comment:
Nit: duplicated lines

Reply:
Thanks for pointing it out.

Removed the duplicate lines in this commit:
https://github.com/geomodulus/torontobot/pull/2/commits/21e37db36f3b27a86cac313ceb56bb84ccdcb45e